docs/source/special.rst

```diff
@@ -44,3 +44,4 @@ Functions
 .. autofunction:: xlog1py
 .. autofunction:: xlogy
 .. autofunction:: zeta
+.. autofunction:: scaled_modified_bessel_i0
```

Comment:
nice rst update although it will likely conflict with the gamma pr (random or alphabetical insertion makes conflicts less likely)

Reply:
Yep. Docs PR forthcoming!